tests/testthat/_snaps/pkgdown_footer.md

```diff
@@ -16,10 +16,10 @@
       pkgdown_footer(data, pkg)
     Output
       $left
-      [1] "Developed by bla. <strong><em>Wow</em></strong>"
+      [1] "<p>Developed by bla.</p><p><strong><em>Wow</em></strong></p>"
```

Comment:
I don't think you want to change behaviour here?

Reply:
I'm a bit undecided. If we normally have inline elements, but let users pass block elements, then the style might be broken for block elements?

Reply:
Either way, you shouldn't change in this PR because it's about refactoring `markdown_text2()` not changing the behaviour of the footer.